Sources/tasks/CreatePost-Notify.php

```diff
@@ -528,8 +528,8 @@ protected function handleWatchedNotifications()
 					(!empty($this->prefs[$member_id]['board_notify']) ? $this->prefs[$member_id]['board_notify'] : 0);
 
 				$content_type = 'board';
-
-				$message_type = !empty($frequency) ? 'notify_boards_once' : 'notify_boards';
+				
```

Comment:
Please remove these unnecessary whitespace characters.

Reply:
Got it. I was sloppy by grabbing a whitespace line for copy/paste from my 2.1.2 working version. Latest commit only changes line 532.